projects/Mallard/src/hooks/use-net-info.tsx

```diff
@@ -1,22 +1,70 @@
-import React, { createContext, useContext } from 'react'
+import React, { createContext, useContext, useMemo } from 'react'
 import {
     useNetInfo as originalUseNetInfo,
     NetInfoState,
     NetInfoStateType,
 } from '@react-native-community/netinfo'
+import { useQuery } from '@apollo/react-hooks'
```

Comment:
ditto, use our own version in `hooks/apollo.ts`

Reply:
As above